src/Tasks.js

```diff
@@ -0,0 +1,33 @@
+export default class Tasks {
+  static index = 0;
+
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  addTask = (description, completed) => {
+    this.tasks.push({
+      description,
+      completed,
+      index: parseInt(Tasks.index, 10),
+    });
+  };
+
+  deleteTask = (index) => {
+    this.tasks.forEach((val, i) => {
+      if (this.tasks[i].index === index) {
+        this.tasks.splice(i, 1);
+      }
+      Tasks.index -= 1;
+    });
+  };
```

Comment:
- [ ] This function should delete a selected task and it should update all remaining items' `indexes`, so they represent the current list order but this is what happens when I delete a task 
here is a list of tasks I have
![delete](https://user-images.githubusercontent.com/44745855/165820762-0f1c697c-ea88-41b0-bec0-558ee6a5987a.png)
see what happens when I try to delete `shopping`
![sh](https://user-images.githubusercontent.com/44745855/165820927-9bcf19d8-3a41-4251-8204-ec2e8a489579.png)
it actually doesn't delete a selected one it deletes others and then return the selected task twice please address this issue